mgr/src/server/plugins/backup_and_restore.cr

```
@@ -0,0 +1,47 @@
+require "moana_types"
+require "file_utils"
+require "../conf"
+require "./helpers"
+require "../datastore/*"
+require "./ping"
+require "./volume_utils.cr"
+
+post "/api/v1/backup" do |env|
+  backupdir = String.from_json(env.request.body.not_nil!)
+
+  Dir.mkdir_p backupdir
+
+  # Copy /var/lib/kadalu/info file into $(backupdir), only if it exists.
+  begin
+    FileUtils.cp("/var/lib/kadalu/info", backupdir)
+  rescue File::NotFoundError
+    halt(env, status_code: 403, response: ({"error": "Info file not found to backup"}.to_json))
```

Ignore this error. Do not halt.

---

You can replace this block with

```
FileUtils.cp("/var/lib/kadalu/info", backupdir) if File.exists?("/var/lib/kadalu/info")
```

---

Done